Documentation/ApiOverview/ContentElements/_AddingYourOwnContentElements/_setup_2.typoscript

```diff
@@ -1,6 +1,6 @@
 tt_content {
-  myextension_newcontentelement =< lib.contentElement
-  myextension_newcontentelement {
-    templateName = NewContentElement
+  examples_basiccontent =< lib.contentElement
```

Comment:
Does not match example key: `examples_newcontentelement`

Reply:
Unified them